db/migrate/20221026062011_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.string :bio
+      t.integer :postsCounter
```

Comment:
- [ ] According to Ruby conventions kindly use underscore for combined names: `posts_counter`